client/src/services/informationManager.ts

```diff
@@ -11,22 +11,78 @@ export const InformationKey = 'information';
 
 export enum InformationDataType {
   WorkspaceRoleChanged,
-  // Add specific events here
+  NewWorkspaceAccess,
+  UserJoinWorkspace,
+  MultipleUsersJoinWorkspace,
+  UserSharedDocument,
+  AllImportedElements,
+  NewDevice,
 }
 
 export interface AbstractInformationData {
   type: InformationDataType;
 }
 
-// Left as an example of how to add additional information type
 export interface WorkspaceRoleChangedData extends AbstractInformationData {
   type: InformationDataType.WorkspaceRoleChanged;
   workspaceId: WorkspaceID;
   oldRole: WorkspaceRole;
   newRole: WorkspaceRole;
 }
 
-export type InformationData = WorkspaceRoleChangedData;
+// The account owner has been granted access to a workspace
+export interface NewWorkspaceAccessData extends AbstractInformationData {
+  type: InformationDataType.NewWorkspaceAccess;
+  workspaceId: WorkspaceID;
+  role: WorkspaceRole;
+}
+
+// A user has been granted access to a workspace
+export interface UserJoinWorkspaceData extends AbstractInformationData {
+  type: InformationDataType.UserJoinWorkspace;
+  workspaceId: WorkspaceID;
+  role: WorkspaceRole;
+  userId: UserID;
+}
+
+// Multiple users have been granted access to a workspace
+export interface MultipleUsersJoinWorkspaceData extends AbstractInformationData {
```

Comment:
If this includes cases where a user has lost access to a workspace, it should be renamed.

Reply:
As discussed, we didn't want this information.